content/swift/concepts/variables/variables.md

```diff
@@ -12,13 +12,15 @@ CatalogContent:
   - 'paths/build-ios-apps-with-swiftui'
 ---
 
-The Swift programming language allows you to store values in two types of containers, `var` for variables and `let` for constants. As the names imply, a variable's value can change, or be reassigned, later in the program while a constant's value cannot. It is a best practice to use constants as much as possible, this will help prevent you from accidentally reassigning values later on without meaning to.
+The Swift programming language allows values to be stored in two types of containers: `var` for variables and `let` for constants.
 
-> Once a variable or constant's data type has been established, whether explicitly declared or inferred by the compiler, it cannot be changed and the variable name cannot be used for another data type within the same program. In the same manner, a variable cannot be changed into a constant and a constant cannot be converted into a variable.
+As the names imply, a variable's value can change or reassigned later in the program, while a constant's value cannot. It is best practice to use constants as much as possible to prevent accidentally reassigning values later on in a program.
+
+> **Note:** Once a variable or constant's data type has been established, whether explicitly or implicitly (inferred by the compiler) declared, it cannot be changed. In addition, the variable name cannot be used for another data type within the same program. In the same manner, a variable cannot be changed into a constant and vice versa.
 
 ## Declaring and Initializing Variables and Constants
 
-Variables and constants can be declared with or without initialization on the same line, however you cannot declare an uninitialized variable without including the type annotation. This makes sense considering that the compiler, when it looks over the code line-by-line, can't make a determination of the data type at the time of the build. Swift also allows a wide range of characters to be used in constant and variable names including Unicode characters and emojis.
+Variables and constants do not require initilization when declared. However, the variable or constant requires type annotation so thatwhen the compiler reads the code line-by-line, it can determine the data type at the time of the build. A `Use of undeclared type` error will be thrown otherwise. Swift also allows a wide range of characters to be used as constant and variable names including Unicode characters and emojis.
```

Comment:
```suggestion
Variables and constants do not require initialization when declared. However, the variable or constant requires type annotation so that when the compiler reads the code line-by-line, it can determine the data type at the time of the build. A `Use of undeclared type` error will be thrown otherwise. Swift also allows a wide range of characters to be used as constant and variable names including Unicode characters and emojis.
```